### PR TITLE
samples: bluetooth: add sysbuild config

### PR DESCRIPTION
**Pull request by henrla**
Add missing sysbuild configuration for multicore
targets running the iso_combined_bis_and_cis sample.

### CODE REVIEW

**Review by hermabe — APPROVED** [review]

**Comment by Tschet1** [comment]
There is a build warning:
_warning: BT_CTLR_SYNC_ISO_STREAM_COUNT (defined at subsys/bluetooth/controller/Kconfig:853) was
assigned the value '1' but got the value ''. Check these unsatisfied dependencies: BT_CTLR_SYNC_ISO
(=n). See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_BT_CTLR_SYNC_ISO_STREAM_COUNT
and/or look up BT_CTLR_SYNC_ISO_STREAM_COUNT in the menuconfig/guiconfig interface. The Application
Development Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of
the manual might be helpful too._

**Comment by hermabe** [comment]
> There is a build warning: _warning: BT_CTLR_SYNC_ISO_STREAM_COUNT (defined at subsys/bluetooth/controller/Kconfig:853) was assigned the value '1' but got the value ''. Check these unsatisfied dependencies: BT_CTLR_SYNC_ISO (=n). See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_BT_CTLR_SYNC_ISO_STREAM_COUNT and/or look up BT_CTLR_SYNC_ISO_STREAM_COUNT in the menuconfig/guiconfig interface. The Application Development Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful too._

On which core? This is expected on the 53 app core since the controller configs dont make sense there.

**Comment by Tschet1** [comment]
> There is a build warning: _warning: BT_CTLR_SYNC_ISO_STREAM_COUNT (defined at subsys/bluetooth/controller/Kconfig:853) was assigned the value '1' but got the value ''. Check these unsatisfied dependencies: BT_CTLR_SYNC_ISO (=n). See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_BT_CTLR_SYNC_ISO_STREAM_COUNT and/or look up BT_CTLR_SYNC_ISO_STREAM_COUNT in the menuconfig/guiconfig interface. The Application Development Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful too._

Tests seem to pass also with this warning.

**Comment by henrla** [comment]
> > There is a build warning: _warning: BT_CTLR_SYNC_ISO_STREAM_COUNT (defined at subsys/bluetooth/controller/Kconfig:853) was assigned the value '1' but got the value ''. Check these unsatisfied dependencies: BT_CTLR_SYNC_ISO (=n). See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_BT_CTLR_SYNC_ISO_STREAM_COUNT and/or look up BT_CTLR_SYNC_ISO_STREAM_COUNT in the menuconfig/guiconfig interface. The Application Development Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful too._
> 
> On which core? This is expected on the 53 app core since the controller configs dont make sense there.

I experience it on the app core, not the net core.

File: samples/bluetooth/iso_combined_bis_and_cis/sysbuild/hci_ipc/prj.conf [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_IPC_SERVICE=y
```
[the remark this answers is not in the document]
Comment (Tschet1) [reply samples/bluetooth/iso_combined_bis_and_cis/sysbuild/hci_ipc/prj.conf:7]: Never mind, the Kconfig.sysbuild would need to be changed to
```suggestion
#
# Copyright (c) 2023 Nordic Semiconductor
#
# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
#

source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"

config NETCORE_IPC_RADIO_BT_HCI_IPC
	default y

config NRF_DEFAULT_IPC_RADIO
	default y
```
(i.e. also remove the NRF_DEFAULT_BLUETOOTH part).
However, not sure what the implications of that would be.

**Review by Tschet1 — APPROVED** [review]

**Pull request merged**